config.libsonnet

```diff
@@ -7,6 +7,10 @@
     clusterUtilizationAlertTime: '5m',
     clusterStateAlertTime: '10m',
 
+    // Constants
+    clusterName: '{{$namespace}}',
```

Comment:
where is $namespace object coming from? and why clusterName is equivalent  to $namepace?

Reply:
May be `StorageType` could be used for filtering the alerts from storage in UI. @cloudbehl ??

Reply:
In service monitor actually rook cluster's name is set equal to namespace. Thinking if the same could be used for naming the cluster as well in UI. @umangachapagain comments?

Reply:
> In service monitor actually rook cluster's name is set equal to namespace. Thinking if the same could be used for naming the cluster as well in UI. @umangachapagain comments?

By default `rook_cluster` label is set to `NAMESPACE` in which it is deployed. And assuming there can be only one cluster in this namespace, we can use NAMESPACE to identify cluster unless we get `Cluster_ID`. 

Looks like discussion on how cluster should be named is going on.

Reply:
As discussed we will remove the cluster_name for the time being. Later if we have cluster_id label, we will use it.